drivers/gles3/storage/texture_storage.cpp

```diff
@@ -1553,6 +1553,14 @@ void TextureStorage::render_target_clear_used(RID p_render_target) {
 	rt->used_in_frame = false;
 }
 
+void TextureStorage::render_target_set_msaa(RID p_render_target, RS::ViewportMSAA p_msaa) {
+	RenderTarget *rt = render_target_owner.get_or_null(p_render_target);
+	ERR_FAIL_COND(!rt);
+	rt->msaa = p_msaa;
+	// TODO: Currently not implemented
```

Comment:
is this TODO real or just a leftover?

Reply:
It's intentional since my initial implementation was only for the Vulkan renderer. But I am currently working on the OpenGL implementation, so I will remove it after I am done :)